tests/SearchUtils.test.js

```diff
@@ -33,6 +33,9 @@ describe('Tests for Search Utils', function() {
 
     beforeAll(function (done) {
         executionPath = path.join(__dirname, 'library');
+        if (!isMac) {
```

Comment:
This is probably a wrong check if we want it to work on Linux in the future. Is the additional `..` required only for Windows only?

Reply:
@VikasShashidhar 
Mac path `executionPath = path.join(__dirname, 'library');`
Windows path `executionPath = path.join(__dirname, '..', 'library');`

In Windows, the library is present in the same dir as the `.exe` path.
But in Mac library is present outside of the dir.

This is the config file for which I'm passing the paths.
![screen shot 2018-02-16 at 1 25 35 pm](https://user-images.githubusercontent.com/596478/36298201-e80b3586-131c-11e8-863f-5960619fb9e0.png)